tests/test_modeling_bart.py

```diff
@@ -382,20 +411,6 @@ def test_dummy_inputs(self):
         model = BartForConditionalGeneration(config).eval().to(torch_device)
         model(**model.dummy_inputs)
 
-    def test_prepare_bart_decoder_inputs(self):
```

Comment:
Deleted the whole `_prepare_bart_decoder_inputs` function which is a private method -> so test is deleted as well